frontend/src/pages/organization/RoleByIDPage/components/OrgRoleModifySection.utils.ts

```diff
@@ -89,7 +89,8 @@ const machineIdentityAuthTemplatePermissionSchema = z
     [OrgPermissionMachineIdentityAuthTemplateActions.EditTemplates]: z.boolean().optional(),
     [OrgPermissionMachineIdentityAuthTemplateActions.DeleteTemplates]: z.boolean().optional(),
     [OrgPermissionMachineIdentityAuthTemplateActions.CreateTemplates]: z.boolean().optional(),
-    [OrgPermissionMachineIdentityAuthTemplateActions.UnlinkTemplates]: z.boolean().optional()
+    [OrgPermissionMachineIdentityAuthTemplateActions.UnlinkTemplates]: z.boolean().optional(),
+    [OrgPermissionMachineIdentityAuthTemplateActions.UseTemplates]: z.boolean().optional()
```

Comment:
minor but can we please update this to be `AttachTemplates`? 

we want to be consistent with what we display in the UI and what we have in the API...

Reply:
please update the doc accordingly as well